cypress/README.md

```diff
@@ -1,6 +1,6 @@
 # Run cypress locally
 
-You can use `./scripts/run-test.sh` to run tests locally. Run it to get usage help.
+You can use `./scripts/test.sh` to run tests locally. Run it to get usage help. You can either let the script run cypress tests by itself, or just use it to quickly prepare your local env with test-specific env/fixtures and do manual tests.
 
 The script will error out if it doesn’t find the necessary env vars. Ask a teammate for specific env vars and put the values in your `.env`.
 
```

Comment:
I don't get the scenario where you are still running the dev env on port 3000 as the entire database will be reseted and the docker instances are run from within the script. IMO you'll have to shutdown the local dev instance no matter what.